### ISSUE TITLE
label-content-name-mismatch incorrectly fails close button with 

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4386]
This should not fail the label-content-name-mismatch rule:

```html
<button aria-label="Close">×</button>
```

I think sort of generally this rule should not fail on single character content elements. For this case I think the "×" needs to be added to the list of chars never to fail. I'm not sure why it isn't already. It would be good to go over the list of unicode characters we're always allowing. There are clearly some gaps there.

More generally, even if we encounter single-character texts that are not on the list it would be better to incomplete them instead of failing. We do a similar thing in the color-contrast rule just to be on the safe side.
